microsoft/testsuites/vm_extensions/linux_patch_extension.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+
+from assertpy.assertpy import assert_that
+from lisa.base_tools.service import Service
+from lisa.sut_orchestrator.azure.common import (
+    get_compute_client,
+    get_node_context,
+    wait_operation,
+)
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+
+from lisa.sut_orchestrator.azure.platform_ import AzurePlatform
+from lisa.util import LisaException
+
+def _verify_vm_agent_running(node: Node, log: Logger) -> None:
+    service = node.tools[Service]
+    is_vm_agent_running = service.is_service_running("walinuxagent.service") or service.is_service_running("waagent.service")
+    
+    if is_vm_agent_running:
+        log.debug("verify walinuxagent or waagent is running")
+        assert_that(is_vm_agent_running).described_as(
+            "Expected walinuxagent or waagent is running"
+        ).is_true()
+    else:
+        log.debug("verify walinuxagent or waagent is not running")
+        raise LisaException("walinuxagent or waagent is not running. Test case failed.")
+
+@TestSuiteMetadata(
+    area="vm_extension",
+    category="functional",
+    description="Test for Linux Patch Extension",
+    requirement=simple_requirement(unsupported_os=[]),
+)
+
+class LinuxPatchExtensionBVT(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        Verify walinuxagent or waagent is running on virtual machine.
+        Perform assess patches to trigger Microsoft.CPlat.Core.LinuxPatchExtension creation in a virtual machine.
+        Verify status file response for validity.
+        """,
+        priority=0,
```

Comment:
set priority as 1+, details please refer to [test-priority](https://mslisa.readthedocs.io/en/main/run_test/microsoft_tests.html#test-priority)

Reply:
done

Reply:
addressed